src/Core/Migration/V6_5/Migration1664541794AddIndexForTasks.php

```diff
@@ -0,0 +1,25 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration\V6_5;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Migration\MigrationStep;
+
+class Migration1664541794AddIndexForTasks extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1664541794;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $connection->executeStatement('ALTER TABLE `cart` ADD INDEX `idx.cart.created_at_updated_at` (`created_at`, `updated_at`)');
```

Comment:
Please ensure that the migration can be executed twice, therefor you should check if the index already exists on the table and only add it if not

Additionally please provide a test case for the migration in the `tests/migration/Core` folder

Reply:
I added a catch. Saw this was done in other index migrations as well.
Unfortunately i'm not familiar with PHPUnit.

Reply:
You can take a look at the existing `\Shopware\Tests\Migration\Core\V6_4\Migration1661759290AddDateAndCurrencyIndexToOrderTableTest` , that test should be pretty easy to adapt to your changes

Reply:
Thnx! I added the test